scripts/check_md5sums.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
```

Comment:
What about `set -eo pipefail`. There's a good long pipe further down and we want to make sure its failure is known.

Reply:
Fair point. I've changed it to that and retested and it's fine.